S/Serd/build_tarballs.jl

```diff
@@ -23,29 +25,16 @@ exit
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [
-    Platform("i686", "linux"; libc = "glibc"),
-    Platform("x86_64", "linux"; libc = "glibc"),
-    Platform("aarch64", "linux"; libc = "glibc"),
-    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
-    Platform("powerpc64le", "linux"; libc = "glibc"),
-    Platform("i686", "linux"; libc = "musl"),
-    Platform("x86_64", "linux"; libc = "musl"),
-    Platform("aarch64", "linux"; libc = "musl"),
-    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
-    Platform("x86_64", "macos"; ),
-    Platform("x86_64", "freebsd"; )
-]
-
+platforms = supported_platforms(; experimental=true)
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("libserd-0", :libserd)
+    LibraryProduct(["libserd-0", "serd-0"], :libserd)
```

Comment:
https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct
```julia
julia> using Base.BinaryPlatforms

julia> parse_dl_name_version("serd-0.dll", "windows")[1]
"serd"
```
```suggestion
    LibraryProduct(["libserd-0", "serd"], :libserd)
```